python/setup.py

```diff
@@ -72,7 +68,7 @@
 setup(
     name='interpret-community',
 
-    version=VERSION,
+    version=interpret_community.__version__,
```

Comment:
Interesting. Curious why is this done this way ?

Reply:
Before, we'd never needed to get the version from the package before. The only place the version was used was in the setup.py file, so that's where we set it. Part of the MLflow process, though, is that the key parts of the environment are stored with their versions in a YAML. To be able to get the version from the package, the place where we store the version had to shift to be inside the package, rather than in the setup.py. setup.py still needs the version, though, so we end up with what you see here.